app/views/settings/save_and_return/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= mojf_settings_screen(
+  heading:t('settings.save_and_return.heading'),
+  description: t('settings.save_and_return.description')
+  ) do |c| %>
+
+  <% c.with_back_link(href: settings_path) %>
+
+<p><%= t('settings.save_and_return.page_lede', href: t('settings.save_and_return.href', url: t('settings.save_and_return.url'))).html_safe %></p>
```

Comment:
if this is a link shouldn't it be an <a> tag? with my other comments about the html not being in the localisation file I'd probably rewrite this to be something like `<a href=<%= settings_path %> ...class ... >t('settings.save_and_return.page_lede')</a>`

Reply:
This is here to embed the url into the paragraph, alternatively, I can remove the localisation and add it all in the template.

Currently we render like this:
```
<p><%= t('settings.save_and_return.page_lede', href: t('settings.save_and_return.href', url: t('settings.save_and_return.url'))).html_safe %></p>
```

But can change to this in the template:
```
<p><%= t('settings.save_and_return.page_lede')%><a class="govuk-link" href="https://moj-forms.service.justice.gov.uk/settings/" target="_blank" rel="noopener noreferrer">user guide section on save for later.</a></p>
```

Both will have the same result, it's just what we'd prefer - I'll leave this comment for now and see if the others have a preference.

Reply:
You can do the second option and still have the `user guide section on save for later.` text as localised. It's just a bit more awkward to inject localised links into localised text but we can't have html in the yaml file IMO, it's not something we could give to a translator and it locks us in to that html